configure.ac

```diff
@@ -355,7 +355,7 @@ AC_SUBST(CFG_CXXFLAGS_PROFILE)
 #_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=gnu++20)
 #_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=c++20)
 case "$(which lsb_release 2>&1 > /dev/null && lsb_release -d)" in
-*Ubuntu*22.04*)
+*Arch*Linux* | *Ubuntu*22.04*)
```

Comment:
For forward compatibility when e.g. C++20 is standard in Arch, you should probably have a version number Like Ubuntu.

Reply:
Arch Linux is a rolling distribution and doesn't have a version number. The output of lsb_release -d contains only the name too.

Reply:
Maybe we should then have a second test for what gcc version?   Not sure if needed.... Do you know what other packages do for this?

Reply:
Ok, let's leave it as proposed, I'll commit it.